modmail/bot/extensions/mail.py

```diff
@@ -3,10 +3,11 @@
 
 import discord
 from discord.ext import commands
+import random as r
 
 import modmail
 from modmail import Config
-from modmail.utils import chron
+from modmail.utils import chron, string, colours
```

Comment:
You should sort these imports. You use VSCode, so you can just right click and select "Sort Imports". Make to run `black . -l119` afterwards.